net-dialup/freeradius/freeradius-3.0.19.ebuild

```diff
@@ -0,0 +1,243 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_{5,6,7}} )
```

Comment:
Add 3.8

Reply:
Here, my fault. It's a preview for the next release 3.0.20 that support also py3. Here, I must drop the support of py3. I will fix this.

Reply:
Only py2.7 but for the next release 3.0.20 I have a question:

The release 3.0.20 contains configure options to define the right paths of py3 to use in the `rlm_python3` module and the right path for `rlm_python`. FWIU this use case it something where python-single-r1 is applicable only for py3 release but not for py2.7.

Something like this:

```bash
$# configure --with-rlm-python3-lib-dir=${PATH_PY3} --with-rlm-python-lib-dir=${PATH_PY2}
```
So, how we could handle this in ebuild for the 3.0.20 release? Is it right to use python eclass because it is a dependency to only linked libraries of the python release choices? So could be correct use:

```bash
RDEPEND="
    python3? ( dev-lang/python:${PYTHON3_VERSION_CHOICE} )
    python? ( dev-lang/python:2.7 )
```
Thanks in advance for any suggestion.